src/main/java/com/artipie/asto/fs/FileStorage.java

```diff
@@ -245,32 +208,6 @@ public FileStorage(final Path path, final ExecutorService exec) {
         return new UnderLockOperation<>(new StorageLock(this, key), operation).perform(this);
     }
 
-    /**
-     * Moves file from source path to destination.
-     *
-     * @param source Source path.
-     * @param dest Destination path.
-     * @return Completion of moving file.
-     */
-    private CompletableFuture<Void> move(final Path source, final Path dest) {
-        return CompletableFuture.supplyAsync(
-            () -> {
-                dest.getParent().toFile().mkdirs();
-                return dest;
-            },
-            this.exec
-        ).thenAcceptAsync(
-            dst -> {
-                try {
-                    Files.move(source, dst, StandardCopyOption.REPLACE_EXISTING);
-                } catch (final IOException iex) {
-                    throw new UncheckedIOException(iex);
-                }
-            },
-            this.exec
-        );
-    }
```

Comment:
@Sammers21 this method probably should be brought back and reused in `save` and `move` methods in order to make `save` atomic again